perllib/FixMyStreet/App/Form/Waste/Bulky.pm

```diff
@@ -110,6 +115,15 @@ has_field resident => (
     ],
 );
 
+has_field tandc => (
```

Comment:
Did this get removed in a previous PR? I do not know why it is showing here as a new addition.